clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -5977,16 +5977,29 @@ void Clang::ConstructJob(Compilation &C, const JobAction &JA,
     // Inputs[1].  Include the header with -include
     if (!IsSYCLOffloadDevice && SYCLDeviceInput) {
       SmallString<128> RealPath;
+#if defined(_WIN32) || defined(_WIN64)
```

Comment:
```suggestion
#if defined(_WIN32)
```
`_WIN32` is predefined for both 32-bit and 64-bit targets.

Reply:
Done, thanks for pointing out